waagent

```diff
@@ -5876,7 +5880,7 @@ def main():
     if len(sys.argv) == 1:
         sys.exit(Usage())
 
-    LoggerInit('/var/log/waagent.log','/dev/console')
+    LoggerInit('/dev/stdout',None)
```

Comment:
The purpose was to make less assumptions about the environment (like availability of `/var/log/waagent.log` or `/dev/console`). These settings are later read from the config and set using `myLogger` var. Default behaviour doesn't change, because these settings are the same in the config file (`Logs.File=/var/log/waagent.log` and `Logs.Console=y`). 

Background: Docker container main process usually doesn't log to a file or serial console. Instead, they just log to stdout and docker decides what to do with it (e.g. log to syslog or you can read these logs with `docker logs` command).

If it does break tests, I want to know which ones, so I can fix.